### ISSUE TITLE
An alternative browser to Falkon or Simple Browser.

### ISSUE THREAD

**Issue by alphamodh0** [helloSystem/hello#270]
https://github.com/OtterBrowser/otter-browser

**Comment by probonopd** [comment]
Hello @alphamodh0. Mind to give some context, such as pros and cons? Thanks.

**Comment by alphamodh0** [comment]
-No KDE dependencies.
-A fully modificable browser.
-Themizable.
-An Opera's 12.x clone.
-Very good for new and advanced users.

**Comment by probonopd** [comment]
> No KDE dependencies.

Hard. Since we are using KWin, there will probably always be _some_ KDE dependencies, which I think is OK (as long as we are not installing "half of KDE Plasma", which we aren't).

> A fully modificable browser.

We have Chromium and Firefox in addition to the system default browser, Falkon. Do these count?

I also have Vivaldi, Brave, Opera, and Ungoogled Chromium running via the Linuxulator on my development machine.

> Themizable.

Probably not going to happen, in order to make all helloSystem computers look and work the same way.

> An Opera's 12.x clone.

Why not Opera itself? :-)

![image](https://user-images.githubusercontent.com/2480569/145468807-35947502-9543-4daf-ae29-acaa9da5351a.png)

Well, it doesn't look, behave, or feel native. For example, the window close button is on the wrong side and it doesn't use the system-provided window title nor the global menu.

> Very good for new and advanced users.

How do we know that we are 'there'?

**Comment by alphamodh0** [comment]
I think the only thing Falkon is missing are "extensions."

**Comment by probonopd** [comment]
That's true. Is there a Qt based browser that supports Chrome or Firefox extensions?

**Comment by alphamodh0** [comment]
> That's true. Is there a Qt based browser that supports Chrome or Firefox extensions?

I don't know if there are any, it would be great.

**Comment by louies0623** [comment]
How about the Pale Moon browser, although it is independent from firefox.
